articles/mdm-migration.md

```diff
@@ -2,46 +2,174 @@
+#### End user workflow
+
+> Available in Fleet Premium
 
-> Automatic enrollment is available in Fleet Premium or Ultimate
+To watch an animation of the end user experience during the migration workflow, in the Fleet UI, head to **Settings > Integrations > Mobile device management (MDM)**, and scroll down to the **End user migration workflow** section.
+
+In Fleet, you can configure the end user workflow using the Fleet UI or fleetctl command-line tool.
+
+Fleet UI:
+
+1. Select the avatar on the right side of the top navigation and select **Settings > Integrations > Mobile device management (MDM)**.
+
+2. Scroll down to the **End user migration workflow** section and select the toggle to enable the workflow.
+
+3. Under **Mode** choose a mode and enter the webhook URL for your automation tool (ex. Tines) under **Webhook URL** and select **Save**.
+
+4. During the end user migration workflow, an end user's device will have their selected system theme (light or dark) applied. If your logo is not easy to see on both light and dark backgrounds, you can optionally set a logo for each theme:
+Head to **Settings** > **Organization settings** >
+**Organization info**, add URLs to your logos in the **Organization avatar URL (for dark backgrounds)** and **Organization avatar URL (for light backgrounds)** fields, and select **Save**.
+
+fleetctl CLI:
+
+1. Create `fleet-config.yaml` file or add to your existing `config` YAML file:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  mdm:
+    macos_migration:
+      enable: true
+      mode: "voluntary"
+      webhook_url: "https://example.com"
+  ...
+```
+
+2. Fill in the above keys under the `mdm.macos_migration` key.
+
+To learn about each option, in the Fleet UI, select the avatar on the right side of the top navigation, select **Settings > Integrations > Mobile device management (MDM)**, and scroll down to the **End user migration workflow** section.
+
+3. During the end user migration workflow, the window will show the Fleet logo on top of a dark and light background (appearance configured by end user).
+
+If you would like to add your organization's logo, you can optionally set a logo for each background:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  org_info:
+    org_logo_url: https://fleetdm.com/images/press-kit/fleet-blue-logo.png
+    org_logo_url_light_background: https://fleetdm.com/images/press-kit/fleet-white-logo.png
+  ...
+```
+
+Add URLs to your logos that are visible on a dark background and light background in the `org_logo_url` and `org_logo_url_light_background` keys respectively. If you only set a logo for one, the Fleet logo will be used for the other.
+
+1. Run the fleetctl `apply -f fleet-config.yml` command to add your configuration.
+
+2. Confirm that your configuration was saved by running `fleetctl get config`.
+
+3. Send [these guided instructions](#how-to-turn-on-mdm-end-user-manual-migration) to your end users to complete the final few steps via Fleet Desktop.
```

Comment:
These steps look a little outdated (they use the old `fleetctl apply` YAML format).

I think let's update these to point to the best practice YAML format in the [YAML files (GitOps) reference docs](https://fleetdm.com/docs/configuration/yaml-files).